client/devdocs/design/index.jsx

```diff
@@ -50,14 +53,22 @@ import ExternalLink from 'components/external-link/docs/example';
 import FeatureGate from 'components/feature-example/docs/example';
 import FilePickers from 'components/file-picker/docs/example';
 import Collection from 'devdocs/design/search-collection';
+import fetchUsageStats from 'state/usage-stats/actions';
 
-export default React.createClass( {
+let DesignAssets = React.createClass( {
 	displayName: 'DesignAssets',
 
 	getInitialState() {
 		return { filter: '' };
 	},
 
+	componentWillMount() {
+		if ( config.isEnabled( 'devdocs/usage-stats' ) ) {
+			const { dispatchFetchUsageStats } = this.props;
+			dispatchFetchUsageStats();
```

Comment:
Oh, I see now. Yeah, I agree that it’s not worth the hassle – this is not crucial information and I doubt somebody leaves this page open for days at a time.

What do you think will be the difference if we fetch the stats in the controller and not in the component?